src/main/java/seedu/address/model/event/StartTime.java

```diff
@@ -83,7 +83,7 @@ public boolean equals(Object other) {
 
     @Override
     public String toString() {
-        return this.time.format(outputFormatter);
+        return String.format("%s:%s", df.format(this.time.getHour()), df.format(this.time.getMinute()));
```

Comment:
Looks like `toString()` returns the same output as `toLogFormat()`. Perhaps set `toString()` to call `toLogFormat()` instead to reduce code duplication?

```suggestion
        return toLogFormat();
```